tests/test_framework/coordinatord.py

```diff
@@ -28,7 +28,7 @@ def __init__(
             "..",
             "servers",
             "coordinatord",
-            "target/debug/revault_coordinatord",
+            "target/debug/main",
```

Comment:
Not to be changed now but it should definitely be called `coordinatord`..

Reply:
https://github.com/revault/coordinatord/issues/36